cmd/runtimetest/main.go

```diff
@@ -82,10 +82,6 @@ func loadSpecConfig(path string) (spec *rspec.Spec, err error) {
 }
 
 func validateUser(spec *rspec.Spec) error {
```

Comment:
This should be `validatePosixUser` to make it extra-clear that it's not covering [Windows users][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#windows-user